lib/components/resinos/default.js

```diff
@@ -48,7 +48,7 @@ exports.injectNetworkConfiguration = (image, configuration) => {
     '[wifi]',
     'hidden=true',
     'mode=infrastructure',
-    `ssid=${configuration.wifiSsid}`,
+    `ssid=${configuration.wifiSSID}`,
```

Comment:
@jviotti if you think it's better to use the camel case, I will remove this one.

Reply:
Its not a matter of preference. The `configuration` object is passed to `resin.models.os.getConfig()` so it depends on what properties that function accepts.